spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -34,64 +34,22 @@
   let(:code_challenge_method) { nil }
 
   describe '#submit' do
-    let(:user) { create(:user) }
-    let(:rails_session_id) { SecureRandom.hex }
-    subject(:result) { form.submit(user, rails_session_id) }
+    subject(:result) { form.submit }
 
     context 'with valid params' do
       it 'is successful' do
         allow(FormResponse).to receive(:new)
 
-        form.submit(user, rails_session_id)
-
-        identity = user.identities.where(service_provider: client_id).first
+        form.submit
 
         extra_attributes = {
           client_id: client_id,
-          redirect_uri: "#{redirect_uri}?code=#{identity.session_uuid}&state=#{state}",
+          redirect_uri: nil,
```

Comment:
The `redirect_uri` key is what is used by the controller to determine whether to redirect or render the error page.